src/navigators.js

```diff
@@ -110,6 +110,7 @@ import DisplayResult from "./screens/ImportAccounts/DisplayResult";
 import FallBackCameraScreen from "./screens/ImportAccounts/FallBackCameraScreen";
 import OnboardingOrNavigator from "./screens/OnboardingOrNavigator";
 import AdvancedLogs from "./screens/AccountSettings/AdvancedLogs";
+import DistributionScreen from "./screens/Distribution";
```

Comment:
(minor. can be named Distribution in the first place)

Reply:
Comes from the fact that I had that superflous stack, changed